lightning/src/ln/channelmanager.rs

```diff
@@ -602,6 +602,29 @@ const CHECK_CLTV_EXPIRY_SANITY: u32 = MIN_CLTV_EXPIRY_DELTA as u32 - LATENCY_GRA
 #[allow(dead_code)]
 const CHECK_CLTV_EXPIRY_SANITY_2: u32 = MIN_CLTV_EXPIRY_DELTA as u32 - LATENCY_GRACE_PERIOD_BLOCKS - 2*CLTV_CLAIM_BUFFER;
 
+/// Channel parameters which apply to our counterparty. These are split out from [`ChannelDetails`]
+/// to better separate parameters.
+#[derive(Clone, Debug, PartialEq)]
+pub struct ChannelCounterpartyParameters {
```

Comment:
While I don't particularly like suffixing with `Details`, I think using that rather than `Parameters` is more accurate and consistent with `ChannelDetails`. When I think about parameters, I think of something that we are passing as a means of configuration. But here these are attributes of the channel specific to the counterparty.

I might even be inclined to use `Attributes` rather than `Details` in both names, but I'd be fine with sticking with `Details`.

Reply:
What about just `ChannelCounterparty`?

Reply:
Works for me